crowbar_framework/app/controllers/reset/nodes_controller.rb

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright 2011-2013, Dell
+# Copyright 2013-2015, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Reset
+  class NodesController < ApplicationController
+    def create
+      @reset = Reset::Nodes.new(
+        valid_params
+      )
+
+      respond_to do |format|
+        if @reset.save
+          format.html do
+            redirect_to root_url
+          end
+          format.json do
+            head :ok
+          end
+        else
+          format.html do
+            flash[:alert] = I18n.t(
+              "reset.nodes.create.reset_failed"
+            )
+
+            redirect_to root_url
+          end
+          format.json do
+            render json: {
+              error: @reset.errors.full_messages.first
+            }, status: :unprocessable_entity
+          end
+        end
+      end
+    end
+
+    protected
+
+    def valid_params
+      params.permit
```

Comment:
Is it possible to mention the explicit params? Like you did in the proposals controller?